src/map.config.json

```diff
@@ -117,20 +164,28 @@
       }
     },
     {
-      "name": "trambusLines",
-      "url": "https://public.sig.rennesmetropole.fr/geoserver/ows",
-      "type": "WMSLayer",
-      "activeOnStartup": true,
-      "layers": "trp_coll:trambus_lignes",
-      "parameters": "format=image/png&transparent=true"
-    },
-    {
-      "name": "trambusStops",
-      "url": "https://public.sig.rennesmetropole.fr/geoserver/ows",
-      "type": "WMSLayer",
+      "name": "poi",
+      "url": "/src/assets/data/POI.json",
```

Comment:
Since it is data, should we put it outside the repo?

Reply:
yes we can do that